arches/app/models/resource.py

```diff
@@ -73,6 +75,14 @@ def __init__(self, *args, **kwargs):
         # end from models.ResourceInstance
         self.tiles = []
         self.descriptor_function = None
+        try:
+            self.serialized_graph = (
+                models.PublishedGraph.objects.filter(publication=self.graph.publication.publicationid, language=settings.LANGUAGE_CODE)
+                .first()
+                .serialized_graph
+            )
+        except:
```

Comment:
I think it would just an AttributeError that would be expected here in the event that no published graph was returned or if self did not have a `serialized_graph` property

Reply:
I think that's correct, and I'm fine with making this change - but there's also the chance that the resource. has no graph at initialization and the graph is assigned later - I have seen this a few places in the code.  I think this'll also trigger an AttributeError, but I am not as knowledgeable about the ORM exceptions.